packages/jazz-tools/src/tools/tests/schema.withPermissions.test.ts

```diff
@@ -688,6 +803,29 @@ describe("setDefaultSchemaPermissions", () => {
     expect(map.name.$jazz.owner.$jazz.id).toEqual(map.$jazz.owner.$jazz.id);
   });
 
+  test("only overrides the provided options", async () => {
+    const anotherAccount = await createJazzTestAccount();
+    setDefaultSchemaPermissions({
+      onInlineCreate: "sameAsContainer",
+    });
+    setDefaultSchemaPermissions({
+      onCreate: (newGroup) => {
+        newGroup.addMember(anotherAccount, "reader");
+      },
+    });
+
+    const TestMap = co.map({
+      name: co.plainText(),
+    });
+    const map = TestMap.create({ name: "Hello" });
+    await map.$jazz.waitForSync();
```

Comment:
Not sure if it's a bug, but if I don't wait for sync here I get a `CoValue has no owner` error in the next line